stories/time-picker/time-picker.stories.js

```diff
@@ -234,7 +242,11 @@ export const CozyAndCompactMode = () => `<div class="fd-container" style="height
 
 CozyAndCompactMode.parameters = {
     docs: {
-        iframeHeight: 580
+        iframeHeight: 580,
+        description: {
+            story: `The time picker is  a composition of \`input-group\`, \`popover\`, and \`time\` components to create a time selection interaction. 
```

Comment:
...by adding the \`fd-input--compact\`, \`fd-button--compact\` and \`fd-time--compact\` modifier classes to the building blocks of the component. ...